requirements/edx/base.txt

```diff
@@ -208,7 +208,6 @@ pyuca==1.1
 pyyaml==5.1.2
 git+https://github.com/edx/RecommenderXBlock.git@1.4.1#egg=recommender-xblock==1.4.1
 redis==2.10.6
-reportlab==3.5.26
```

Comment:
This is also in `testing.txt` and `development.txt`; you should probably just run `make upgrade` to update these.

Reply:
I tried running `make upgrade` in both local and the lms container and it kept trying to replace the dependencies for things `common/lib/capa` with the `file://diana/workspace/edx-platform` version of that.